tests/pipelines/test_pipelines_image_segmentation.py

```diff
@@ -229,12 +229,12 @@ def test_small_model_pt_semantic(self):
                 {
                     "score": None,
                     "label": "LABEL_0",
-                    "mask": "775518a7ed09eea888752176c6ba8f38",
+                    "mask": "42d09072282a32da2ac77375a4c1280f"
                 },
                 {
                     "score": None,
                     "label": "LABEL_1",
-                    "mask": "a12da23a46848128af68c63aa8ba7a02",
+                    "mask": "46b8cc3976732873b219f77a1213c1a5",
```

Comment:
Tests updated to reflect change in masks: 

Input image: 
![input_image](https://user-images.githubusercontent.com/22614925/195845867-64ef155f-5664-4fe9-a36b-8fa525f0800e.png)

Masks on `main`: 

![main_label_0](https://user-images.githubusercontent.com/22614925/195845897-3adf4684-b986-4fb9-abc9-b5fe07a357b7.png)
![main_label_1](https://user-images.githubusercontent.com/22614925/195845900-091a49cb-fd2f-4a1b-a929-4140eab91e2d.png)




Masks on this branch: 

![fix_label_0](https://user-images.githubusercontent.com/22614925/195845919-a3bfff63-4b4f-413e-8373-cf8fd9269a90.png)
![fix_label_1](https://user-images.githubusercontent.com/22614925/195845921-16f843c7-fe3d-4299-b27a-98409d0aa758.png)